apps/masterbots.ai/components/routes/chat/chat-panel.tsx

```
@@ -67,17 +70,17 @@ export function ChatPanel({
-            onSubmit={async (value) => {
+            onSubmit={async value => {
               await append({
                 id,
                 content: value,
-                role: "user",
-              });
+                role: 'user'
+              })
             }}
-            disabled={!Boolean(chatbot)}
+            disabled={!Boolean(chatbot)} // Disables input if no chatbot is provided
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)